app/src/main/java/com/infomaniak/mail/ui/login/IntroFragment.kt

```diff
@@ -109,29 +106,21 @@ class IntroFragment : Fragment() {
         lifecycleScope.launch(Dispatchers.IO) {
             val duration = resources.getInteger(R.integer.loginLayoutAnimationDuration).toLong()
             delay(duration)
-            introViewModel.currentAccentColor.postValue(accentColor)
+            loginViewModel.currentAccentColor.postValue(accentColor)
         }
     }
 
     private fun updateUiWhenThemeChanges(position: Int?) {
-        introViewModel.currentAccentColor.observe(viewLifecycleOwner) { accentColor ->
-            setUi(
-                accentColor = accentColor ?: UiSettings.DEFAULT_ACCENT_COLOR,
-                position = position,
-                animate = accentColor != null,
-            )
+        loginViewModel.currentAccentColor.observeNotNull(viewLifecycleOwner) { accentColor ->
+            setUi(accentColor, position)
         }
     }
 
-    /**
-     * `animate` is necessary because when the activity is started we want to avoid animating the color change the first time.
-     */
-    private fun setUi(accentColor: AccentColor, position: Int?, animate: Boolean = true) = with(binding) {
+    private fun setUi(accentColor: AccentColor, position: Int?) = with(binding) {
+        val animate = position == 0
```

Comment:
This won't prevent animation when adding your first account to the app. Also add back old comment